diff --git a/certbot_dns_godaddy.py b/certbot_dns_godaddy.py
index 594d25c..a3b8f6c 100644
--- a/certbot_dns_godaddy.py
+++ b/certbot_dns_godaddy.py
@@ -6,8 +6,7 @@ removing, TXT records using the godaddy CCP API.
 
 import logging
 
-import zope.interface
-from certbot import errors, interfaces
+from certbot import errors
 from certbot.plugins import dns_common, dns_common_lexicon
 from lexicon.providers import godaddy
 from requests.exceptions import RequestException
@@ -16,8 +15,6 @@ import tldextract
 logger = logging.getLogger(__name__)
 
 
-@zope.interface.implementer(interfaces.IAuthenticator)
-@zope.interface.provider(interfaces.IPluginFactory)
 class Authenticator(dns_common.DNSAuthenticator):
     """DNS Authenticator for godaddy
 
diff --git a/pyproject.toml b/pyproject.toml
index cfe8a66..0ca201b 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -34,7 +34,6 @@ python = ">=3.6,<4.0"
 acme = ">=0.31.0"
 certbot = ">=0.31.0"
 dns-lexicon = ">=3.2.3"
-"zope.interface" = ">=5.4.0"
 urllib3 = "^1.26.4"
 
 [tool.poetry.dev-dependencies]
